src/tests/system/tests/test_memory_cache.py

```diff
@@ -602,7 +604,7 @@ def check():
 @pytest.mark.importance("critical")
 @pytest.mark.cache
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
-def test_memory_cache__getpwnam_fully_qualified_names(client: Client, provider: GenericProvider):
+def test_memory_cache__lookup_users_by_fully_qualified_names(client: Client, provider: GenericProvider):
```

Comment:
later you use "fqn" instead of spelling out "fully_qualified_names".   Could that be used here?

Reply:
I know, the other test case name would be long, I prefer it fully spelled out until the name is too long. I changed the other test case name.